src/test/scala/com/campudus/tableaux/MediaTest.scala

```diff
@@ -217,6 +223,142 @@ class MediaTest extends TableauxTestBase {
     }
   }
 
+  @Test
+  def createAttachmentColumn(): Unit = okTest {
+    val expectedJson = Json.obj("status" -> "ok", "columns" -> Json.arr(Json.obj("id" -> 3, "ordering" -> 3)))
+
+    val column = Json.obj("columns" -> Json.arr(Json.obj(
+      "kind" -> "attachment",
+      "name" -> "Downloads"
+    )))
+
+    for {
+      tableId <- setupDefaultTable()
+
+      column <- sendRequestWithJson("POST", column, s"/tables/$tableId/columns")
+    } yield {
+      assertEquals(expectedJson, column)
+    }
+  }
+
+  @Test
+  def fillAndRetrieveAttachmentCell(): Unit = okTest {
+    val expectedJson = Json.obj("status" -> "ok", "columns" -> Json.arr(Json.obj("id" -> 3, "ordering" -> 3)))
+
+    val column = Json.obj("columns" -> Json.arr(Json.obj(
+      "kind" -> "attachment",
+      "name" -> "Downloads"
+    )))
+
+    val fileName = "Scr$en Shot.pdf"
+    val file = s"/com/campudus/tableaux/uploads/$fileName"
+    val mimetype = "application/pdf"
+    val putFile = Json.obj("name" -> "Test PDF", "description" -> "A description about that PDF.")
+
+    for {
+      tableId <- setupDefaultTable()
+
+      columnId <- sendRequestWithJson("POST", column, s"/tables/$tableId/columns") map {
+        json =>
+          json.getArray("columns").get[JsonObject](0).getField[Int]("id")
```

Comment:
can be simplified to `map (_.getArray("columns").get[JsonObject](0).getField[Int]("id"))`